src/sklearn_evaluation/plot/roc.py

```diff
@@ -349,3 +484,39 @@ def _calculate_plotting_data(y_true, y_score):
             tpr = [tpr]
 
         return fpr, tpr, label
+
+
+def _preprocess_array_for_roc(array):
+    """
+    https://github.com/ploomber/sklearn-evaluation/pull/221/files
```

Comment:
is this comment needed here?

Reply:
This method is not in use and @edublancas asked me to keep it so I wanted to refer to the original discussion. I'll remove it and add a better description.